### PR TITLE
fix(compat): replace matchAll to support iOS safari <= 12

### PR DESCRIPTION
**Pull request by kellyjosephprice**
[][demo] |
:---:|

## 🧰 Changes

`matchAll` first came to iOS safari in version 13.

## 🧬 QA & Testing

[demo]: https://markdown-pr-168.herokuapp.com
[prod]: https://SUBDOMAIN.readme.io

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -28,9 +28,9 @@ jobs:
     - name: Run visual regression tests
       run: CI=true npm run test.browser
 
-    - name: Update regression test snapshots
-      if: ${{ failure() }}
-      run: npm run test.browser -- -u
+    #- name: Update regression test snapshots
+      #if: ${{ failure() }}
+      #run: npm run test.browser -- -u
```
Comment (rafegoldberg) [inline .github/workflows/ci.yml]: Don't we need to update these visual snapshots in order to get tests to pass?

File: components/HTMLBlock.jsx [hunk 2]
```diff
@@ -6,7 +6,11 @@ const PropTypes = require('prop-types');
 const MATCH_SCRIPT_TAGS = /<script\b[^>]*>([\s\S]*?)<\/script>\n?/gim;
 
 const extractScripts = (html = '') => {
-  const scripts = [...html.matchAll(MATCH_SCRIPT_TAGS)].map(m => m[1]);
```
Comment (rafegoldberg) [inline components/HTMLBlock.jsx:9]: We should probably polyfill missing native methods like this in the long run.

**Review by rafegoldberg — APPROVED** [review]
Thanks for the quick turnaround here. Looks good to merge once we get those tests fixed!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (kellyjosephprice) [reply .github/workflows/ci.yml]: Yes!

File: components/HTMLBlock.jsx (hunk 2 above)
Comment (kellyjosephprice) [reply components/HTMLBlock.jsx:9]: I don't understand babel, but shouldn't it already be:

```
$ cat .babelrc 
{
  "presets": [
    [
      "@babel/preset-env",
      {
        "targets": {
          "browsers": ["last 2 versions"]
        }
      }
    ],
    "@babel/preset-react"
  ],
  "plugins": [
    "@babel/plugin-proposal-class-properties",
    "@babel/plugin-proposal-object-rest-spread",
    "@babel/plugin-proposal-optional-chaining"
  ]
}
```

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.27.2-next.2</b>](/readmeio/markdown/releases/tag/v6.27.2-next.2)
<br>

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.27.2</b>](/readmeio/markdown/releases/tag/v6.27.2)
<br>

**Pull request merged**